### PR TITLE
feat(e2e/knuu): use latest release and prepare for node knuu tests

### PR DESCRIPTION
**Pull request by smuu**
Closes https://github.com/celestiaorg/celestia-app/issues/4075

## Overview

This PR:
- Uses the latest release of knuu with fixes
- Prepares the testnet to be used by celestia-node

### CODE REVIEW

**Review by rootulp — DISMISSED** [review]

**Review by ninabarbakadze — COMMENTED** [review]
thanks sam! lint needs fixing before it can merge

File: test/e2e/testnet/testnet.go [hunk 1]
```diff
@@ -384,22 +392,24 @@
...
 				}
 			} else {
-				err = errors.New("error getting status")
+				log.Error().Str("name", node.Name).Int("attempt", i).Err(err).Msg(
+					"error getting status, retrying...")
 			}
-			if i == 9 {
-				return fmt.Errorf("failed to start node %s: %w", node.Name, err)
+			if i == 19 {
+				return fmt.Errorf("timed out waiting for node %s to sync: %w", node.Name, err)
```
[the remark this answers is not in the document]
Comment (evan-forbes) [reply test/e2e/testnet/testnet.go:411]: I don't want to block on this @smuu if we're close on time, but these logic statements make maintaining this code more cumbersome than is required

[the remark this answers is not in the document]
Comment (smuu) [reply test/e2e/testnet/testnet.go]: fixed

File: test/e2e/testnet/testnet.go [hunk 2]
```diff
@@ -378,28 +370,25 @@
...
-					log.Info().Int("attempts", i).Str("name", node.Name).Msg(
-						"node has synced")
+				if resp != nil && resp.SyncInfo.LatestBlockHeight > 0 {
+					t.logger.Println("node has synced", "name", node.Name, "attempts", i, "latest_block_height", resp.SyncInfo.LatestBlockHeight)
 					break
 				}
+				t.logger.Println("node status retrieved but not synced yet, waiting...", "name", node.Name, "attempt", i)
 			} else {
-				err = errors.New("error getting status")
+				t.logger.Printf("error getting status, retrying...", "name", node.Name, "attempt", i, "error", err)
```
[the remark this answers is not in the document]
Comment (smuu) [reply test/e2e/testnet/testnet.go]: fixed

File: test/e2e/main.go [hunk 3]
```diff
@@ -56,7 +56,7 @@
 	if err != nil {
 		logger.Fatalf("--- ERROR %s: %v", test.Name, err)
 	}
-	logger.Printf("--- ✅ PASS: %s \n\n", test.Name)
+	logger.Println("--- ✅ PASS: %s \n\n", test.Name)
```
[the remark this answers is not in the document]
Comment (smuu) [reply test/e2e/main.go]: fixed

**Review by evan-forbes — APPROVED** [review]
:+1:

**Review by rootulp — APPROVED** [review]

**Pull request merged**